crypto/s2n_hmac.c

```diff
@@ -258,6 +275,18 @@ int s2n_hmac_digest_verify(const void *a, const void *b, uint32_t len)
 
 int s2n_hmac_copy(struct s2n_hmac_state *to, struct s2n_hmac_state *from)
 {
-    memcpy_check(to, from, sizeof(struct s2n_hmac_state));
+    to->alg = from->alg;
+    to->hash_block_size = from->hash_block_size;
+    to->currently_in_hash_block = from->currently_in_hash_block;
+    to->block_size = from->block_size;
+    to->digest_size = from->digest_size;
+
```

Comment:
Good question! OpenSSL's EVP_Digest API's require (and enforce) the use of their copy API to copy any hash state. They may have some external state that the API is also preserving. I have added a comment to ensure that we remember this.